invenio_deposit/ext.py

```diff
@@ -38,6 +40,8 @@ def __init__(self, app=None):
         """Extension initialization."""
         if app:
             self.init_app(app)
+        post_action.connect(index_deposit_after_publish, sender=app,
```

Comment:
Which will be our case in B2Share as we use a subclass of Deposit, at least until there is another way to extend Deposit.